forc-plugins/forc-client/src/cmd/submit.rs

```diff
@@ -2,9 +2,17 @@ use crate::NodeTarget;
 use devault::Devault;
 use std::path::PathBuf;
 
+forc::cli_examples! {
+    [ Submit a transaction from a json file => submit "./mint.json" => "Submission of tx or awaiting commit failed" ]
```

Comment:
```suggestion
    [ Submit a transaction from a json file => submit "./mint.json" => "Awaiting submission of tx or failed commit" ]
```

Unsure what a failed commit is so I would probably rephrase that too.